processing/src/main/java/io/druid/segment/data/EntireLayoutIndexedLongSupplier.java

```diff
@@ -59,7 +60,7 @@ public void fill(int index, long[] toFill)
     {
       if (totalSize - index < toFill.length) {
         throw new IndexOutOfBoundsException(
-            String.format(
+            StringUtils.safeFormat(
```

Comment:
This pattern is not used too many times and it's not as immediately understandable abbr for Java devs as NPE or ISE, so no